app/models/v2/tailoring.rb

```diff
@@ -39,6 +39,7 @@ class Tailoring < ApplicationRecord
              inverse_of: :tailoring
     has_many :rules, class_name: 'V2::Rule', through: :tailoring_rules
     has_many :reports, class_name: 'V2::Report', dependent: nil
+    has_many :test_results, class_name: 'V2::TestResult', dependent: :destroy
```

Comment:
The `has_many :reports` will not work as it is, it could be tied through the `policy`